src/components/optimizerTab/Sidebar.jsx

```diff
@@ -277,26 +287,26 @@ function SidebarContent() {
                     loading={optimizationInProgress}
                     onClick={startClicked} style={{ flex: 1 }}
                   >
-                    Start optimizer
+                    {t('Sidebar.ControlsGroup.Start')/* Start optimizer */}
                   </Button>
                 </Flex>
 
                 <ComputeEngineSelect/>
 
                 <Flex gap={defaultGap}>
                   <Button onClick={cancelClicked} style={{ flex: 1 }}>
-                    Cancel
+                    {t('Sidebar.ControlsGroup.Cancel')/* Cancel */}
                   </Button>
 
                   <Popconfirm
-                    title='Reset all filters?'
-                    description='All filters will be reset to their default values'
+                    title={t('Sidebar.ControlsGroup.ResetConfirm.Title')}// 'Reset all filters?'
+                    description={t('Sidebar.ControlsGroup.ResetConfirm.Description')}// 'All filters will be reset to their default values'
                     onConfirm={resetClicked}
-                    okText='Yes'
-                    cancelText='No'
+                    okText={t('Sidebar.ControlsGroup.ResetConfirm.Yes')}// 'Yes'
+                    cancelText={t('Sidebar.ControlsGroup.ResetConfirm.No')}// 'No'
                   >
                     <Button style={{ flex: 1 }}>
-                      Reset
+                      {t('Sidebar.ControlsGroup.Reset')/* Reset */}
```

Comment:
Should these be under common?

Reply:
my thought process was to try and avoid directly accessing the common namespace in the code with the exception of certain things like stats. That way any change to the displayed text can be done solely by editing the locale files instead of having to both change the key and add a new entry in the translation files